ale/isd_generate.py

```diff
@@ -84,9 +97,12 @@ def main():
         except KeyError:
             k = [args.kernel, ]
 
+    if args.radius is not None:
+        rad = args.radius
+
```

Comment:
why not omit this and just pass args.radius in on line 105? Also, just reading the code, doesn't this error if you omit a radius because rad is undefined when you hit line 105?

Reply:
I will omit, but no it won't error as the default is set to None.

Reply:
Yes this if-statement is redundant and will lead to an error as described.

Just passing `args.radius` to the `radius=` parameter of `file_to_isd()` is the right pattern here.